README.md

```diff
@@ -1,9 +1,14 @@
-![](https://www.ga4gh.org/wp-content/themes/ga4gh-theme/gfx/GA-logo-horizontal-tag-RGB.svg)
-# GA4GH Discovery - Hash Algorithm Registry [![LICENCE](https://img.shields.io/github/license/susheel/ga4gh-hash-alg-registry)](https://github.com/susheel/ga4gh-hash-alg-registry/blob/master/LICENSE) 
+# GA4GH Discovery - Checksum Specification [![LICENCE](https://img.shields.io/github/license/ga4gh-discovery/ga4gh-checksum)](https://github.com/ga4gh-discovery/ga4gh-checksum/blob/master/LICENSE)
 
-This registry extends the [IANA Hash Algorithm Registry](https://www.iana.org/assignments/named-information/named-information.xhtml#hash-alg) with hash algorithms specific to the GA4GH community requirements.
+Checksums are core component of building reproducible computational workflows on verifying datasets. They are typically fixed size bit-strings generated by applying a checksum algorithm on a block of data to help verify the integrity of the data. The specification provides a reusable schema object definition to specify a checksum algorithm type and value.
 
-## Specification - [Table of Hash Algorithms](https://github.com/susheel/ga4gh-hash-alg-registry/blob/master/hash-alg.csv)
+This repository also hosts the GA4GH Hash Algorithm Registry that extends the [IANA Hash Algorithm Registry](https://www.iana.org/assignments/named-information/named-information.xhtml#hash-alg) with hash algorithms specific to the GA4GH community requirements.
+
+## Specification
+
+Checksum is described as a [SchemaBlock object](https://github.com/ga4gh-discovery/ga4gh-checksum/blob/master/checksum.yaml).
```

Comment:
This doesn't exist ... Probably the best general link would be 

https://schemablocks.org/categories/schemas.html

But you should represent it directly in SB, probably in its own project repository (sb-checksum).

Reply:
@mbaudis I have put this as part of this PR here: https://github.com/ga4gh-discovery/ga4gh-checksum/blob/e21085cb85bb39fec0f49bba667b563850b5a254/checksum.yaml

Could you do a sanity check on the proposed {S}[B] YAML spec above?

Happy to move this into its own SB repo.

Reply:
In principle we have said that - so far - schemas are developed (& approved etc.) in their own WS/DP; and then a versioned release (or state) is used for an {S}[B] re-write. This is chiefly a "non-interference" measure w/ the larger dev. projects, but also for developing operational procedures.
For now I would:

* write such projects like _Checksum_ in their own space (e.g. here...)
* label a version & move this to {S}[B] (i.e. in the newly created [sb-checksum](https://github.com/ga4gh-schemablocks/sb-checksum) repo :-)
* re-write it in {S}[B] JSON Schema flavour to [schemas](https://github.com/ga4gh-schemablocks/sb-checksum/schemas/) there, including correct `$id`
* add it to the `schema_repos` in the [tools config](https://github.com/ga4gh-schemablocks/tools/blob/master/sbSchemaParser/config.yaml/)

At a first glance I would change the `$id` value here (since this is not {S}[B]), and then fix it in the {S}[B] version accordingly - see the new structure including project in the path, e.g.:

```
"$id": https://schemablocks.org/schemas/sb-checksum/Checksum/v0.0.1
```
... or such.